test/topology_experimental_raft/test_raft_service_levels.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import pytest
+import time
+import asyncio
+import logging
+from test.pylib.util import unique_name, wait_for_cql_and_get_hosts
+from test.pylib.manager_client import ManagerClient
+from test.pylib.internal_types import ServerInfo
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement
+
+
+logger = logging.getLogger(__name__)
+
+async def trigger_snapshot(manager: ManagerClient, servers: [ServerInfo]) -> None:
+    for server in servers:
+        cql = manager.get_cql()
+        host = cql.cluster.metadata.get_host(server.ip_addr)
+        group0_id = (await cql.run_async(
+            "select value from system.scylla_local where key = 'raft_group0_id'",
+            host=host))[0].value
```

Comment:
Group 0 ID is the same on all nodes in the cluster, therefore there is no need to fetch it before issuing a barrier on each node.